docs/installation/index.md

```diff
@@ -91,7 +91,7 @@ yum-config-manager --enable rhel-6-server-optional-rpms
 <div id="el6" markdown="1">
 ```bash
 yum -y localinstall http://fedorapeople.org/groups/katello/releases/yum/{{ site.version }}/katello/el6/x86_64/katello-repos-latest.rpm
-yum -y localinstall http://yum.theforeman.org/releases/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
+yum -y localinstall http://fedorapeople.org/groups/katello/releases/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
```

Comment:
I don't see a formean directory here? https://fedorapeople.org/groups/katello/releases/ 

I think you mean https://fedorapeople.org/groups/katello/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm ?